samples/MvcAngular/AngularClient/.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+	"directory" : "app/dist"
```

Comment:
Please remove this directive (well, the whole file actually :smile:), since it causes a totally unwanted behavior with Sake, that will try to call `npm install` in each bower component folder, making the build process extremely long.

You should use grunt to move the JS/CSS files you need from `bower_components` to your `dist` folder.